libogc/exi.c

```diff
@@ -38,6 +38,8 @@ distribution.
 #include "exi.h"
 #include "gcutil.h"
 
+#include "lwp_queue.h"
```

Comment:
how did this header get included before? :/

Reply:
typo haha